systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java

```diff
@@ -634,7 +634,7 @@ void testDynamicallySetClusterOperatorLoggingLevels(ExtensionContext extensionCo
         long reconciliationSleep = RECONCILIATION_INTERVAL + Duration.ofSeconds(10).toMillis();
         LOGGER.info("Waiting {} ms log not to be empty", reconciliationSleep);
         // wait enough time (at least for reconciliation time + 10) and check whether logs after this time are not empty
-        Thread.sleep(reconciliationSleep);
+        Thread.sleep(reconciliationSleep * 2);
```

Comment:
Wouldnt be better to use some dynamic wait?

Reply:
I guess it will be better to change it in all other tests, as we have the `Thread.sleep()` in (almost) every dynamic config test.